cypress/tests/core/basic/logout.js

```diff
@@ -0,0 +1,19 @@
+import { ploneAuth } from '../../../support/constants';
+
+describe('Logout Tests', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains('Log in').click();
+    const user = ploneAuth[0];
+    const password = ploneAuth[1];
+
+    cy.get('#login').type(user).should('have.value', user);
+    cy.get('#password').type(password).should('have.value', password);
+    cy.get('#login-form-submit').click();
+    cy.get('body').should('have.class', 'has-toolbar');
+  });
+  it('As registered user I can logout', function () {
+    cy.get('#toolbar-personal').click();
+    cy.get('#toolbar-logout').click();
```

Comment:
Maybe just assert if we have `auth_token`? 
https://docs.cypress.io/api/commands/getcookie